rgw/v2/tests/s3_swift/test_Mbuckets_with_Nobjects.py

```diff
@@ -332,6 +332,20 @@ def test_exec(config):
                             raise TestExecError(
                                 "Command failed....Bucket is not added into reshard queue"
                             )
+                    if config.bucket_sync_status:
+                        time.sleep(30)
```

Comment:
Shall we remove this delay and manage it in utils files?

Reply:
done.